lib/patchelf/patcher.rb

```diff
@@ -15,11 +16,22 @@ class Patcher
     # Instantiate a {Patcher} object.
     # @param [String] filename
     #   Filename of input ELF.
-    def initialize(filename)
+    # @param [Boolean] logging
+    #   Whether to use a (stderr-initialized) logger for errors
+    def initialize(filename, logging: true)
       @in_file = filename
       @elf = ELFTools::ELFFile.new(File.open(filename))
       @set = {}
       @rpath_sym = :runpath
+      @logging = logging
+    end
+
+    def log_or_raise(msg)
```

Comment:
Feeling this would not be used outside this class, if so please move it to the private methods area

Reply:
Ah! I missed that area. Will fix now!